direct/nn/rim/tests/test_rim.py

```diff
@@ -89,4 +91,16 @@ def test_rim(
 
     out = model(img, kspace, mask, sens)[0][-1]
 
+    ov_model = OpenVINOModel(model)
+    ov_out = ov_model(img, kspace, mask, sens)
```

Comment:
make OpenVINO model output similar to origin Torch output. Produce exactly the same structure (list of outputs i.e.) and wrap to `torch.tensor`. So users will have a smooth integration